src/+otp/+sanzserna/SanzSernaProblem.m

```diff
@@ -0,0 +1,27 @@
+classdef SanzSernaProblem < otp.Problem
```

Comment:
Let's add a link to the paper where we got the problem.  We can also try to think of names other than SanzSerna.